source/backend/routes/users.js

```diff
@@ -86,4 +86,20 @@ router.delete('/:savedRecipeId', async (req, res) => {
   }
 });
 
+/* POST /user/completedRecipes */
+router.post('/completedRecipes', async (req, res) => {
+  try {
+    const { userId } = req.userInfo;
+    const { recipeId } = req.body.recipeId;
```

Comment:
So for line 93, when you use {} for a variable, I think the term is "destructuring." This shortcut is for when you have a dictionary or json format and the field name you are trying to take matches. This is best shown in an example: 
```
myDict = {hello: 'world', goodbye: 'coder'}
// if you want to take the value 'world' and save it into hello
hello = myDict['hello']
// or you can do this
{hello} = myDict
```
This is a bit of an overkill response, but hopefully, this helps understand conceptionally what those brackets do

Reply:
This explanation was perfect, thanks David!